components/Common/Sidebar/index.tsx

```diff
@@ -0,0 +1,44 @@
+import type { Url } from 'url';
+
+import { useState } from 'react';
+import type { FC } from 'react';
+
+import styles from './index.module.css';
+import SidebarGroup from './SidebarGroup';
```

Comment:
```suggestion
import SidebarGroup from '@/components/Common/Sidebar/SidebarGroup';
```

Component imports should be done via alias